src/Entities/Collection.ts

```diff
@@ -44,9 +44,6 @@ export class Collection {
   @Field(type => Boolean, {
     description: "Collection has its description available as markdown",
   })
-  @Column({ default: false })
-  hasMarkdownDescription: boolean = false
-
```

Comment:
Looks like we removed the db column (via the `@Column` decorator) but not the GraphQL field (via the `@Field` decorator)